### PR TITLE
[DPC-4081] Remove .html from internal links

### PR DESCRIPTION
**Pull request by ashley-weaver**
## 🎫 Ticket

https://jira.cms.gov/browse/DPC-4081

## 🛠 Changes

[dpc-ops PR](https://github.com/CMSgov/dpc-ops/pull/698)

## ℹ️ Context for reviewers

(Background context, more in-depth details of the implementation, and anything else you'd like to call out or ask reviewers.)

## ✅ Acceptance Validation

(How were the changes verified? Did you fully test the acceptance criteria in the ticket? Provide reproducible testing instructions and screenshots if applicable.)

## 🔒 Security Implications

- [ ] This PR adds a new software dependency or dependencies.
- [ ] This PR modifies or invalidates one or more of our security controls.
- [ ] This PR stores or transmits data that was not stored or transmitted before.
- [ ] This PR requires additional review of its security implications for other reasons.

If any security implications apply, add Jason Ashbaugh (GitHub username: StewGoin) as a reviewer and do not merge this PR without his approval.

### CODE REVIEW

**Comment by ashley-weaver** [comment]
> Do we also need them removed from the implementation guide? For example: https://dpc.cms.gov/ig/StructureDefinition-dpc-profile-practitioner.html

I wasn't sure, but I can.

File: common/faq.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 title: Frequently Asked Questions
 banner_title: Frequently Asked Questions
 description: Frequently asked questions about the Data at the Point of Care pilot
-permalink: /faq.html
+permalink: /faq
```
Comment (jdettmannnava) [inline common/faq.html:6]: html on lines 26 and 36

**Review by jdettmannnava — COMMENTED** [review]
looks like an html reference in _includes/_hero.html

**Review by jdettmannnava — APPROVED** [review]
LGTM

**Pull request merged**